tools/project_manager/post_files.inc

```diff
@@ -107,13 +107,14 @@ function generate_interim_file($projectid, $limit_round_id, $which_text, $includ
     fclose($handle);
     // make the zip and sent it to the user
     header("Content-type: application/octet-stream");   
-    header("Content-disposition: attachment; filename={$filename}.zip");   
-    passthru("zip -q -j - $textfile_path", $return_code);
-    if ($return_code != 0)
-    {
-        echo "post_files: the zip command failed.";
-    }
-    // cleanup will get called whatever happens  
+    header("Content-disposition: attachment; filename={$filename}.zip");
+
+    $zip_path = "{$dirname}/{$filename}.zip";
+    create_zip_from([$textfile_path], $zip_path);
+    readfile($zip_path);
+    unlink($zip_path);
```

Comment:
Would it be better to put this in `function clean_up_temp($dirname, $textfile_path)` for consistency?

Reply:
Yes, the unlink should be in `clean_up_temp()` too.